platform-coop-toolkit/coop-library-framework
# Configure acceptance testing

## Description

This PR adds experimental acceptance testing using [Cypress](https://www.cypress.io/).

## Steps to test

1. Check out this branch.
2. Run `composer install`.
3. Run `npm install`.
4. Run `npm test`.

**Expected behavior:** Tests pass.

## Additional information

Cypress Docs: https://docs.cypress.io/

## Related issues

Not applicable.
